Specs/Sequence/English/URLModel.json

```diff
@@ -394,5 +394,30 @@
     "Input": "Please visit https://luis.xxx?action=add for more information",
     "NotSupportedByDesign": "javascript, python",
     "Results": []
+  },
+  {
+    "Input": "working..is isn't a valid domain name.",
+    "NotSupportedByDesign": "python",
+    "Results": []
+  },
+  {
+    "Input": "7.am is more likely a datetime string rather than a valid domain name.",
+    "NotSupportedByDesign": "python",
+    "Results": []
+  },
+  {
+    "Input": "Please visit https://7.am for more information.",
```

Comment:
What about something like "27.pm" or "s7.am"? Those could still be valid URLs. Does the current code account for it?